tests/stdlib/tmath.nim

```diff
@@ -1,13 +1,12 @@
 discard """
-  action: run
-  matrix:"; -d:nimTmathCase2 -d:danger --passc:-ffast-math"
+  targets: "c js"
 """
 
-# xxx: fix bugs for js then add: targets:"c js"
+## xxx enable matrix:"; -d:nimTmathCase2 -d:danger --passc:-ffast-math"
```

Comment:
In the worst case, create a tmath2.nim test that has this test matrix and includes tmath.nim

Reply:
the test even doesn't work locally. I don't understand why they can pass the tests before
`nim_temp.exe  c -r -d:nimTmathCase2 -d:danger --passc:-ffast-math tests/stdlib/tmath.nim`

```
fatal.nim(49)            sysFatal
Error: unhandled exception: tmath.nim(204, 15) `classify(trunc(f_neg_one * f_zero)) == fcNegZero`  [AssertionDefect]
```

Reply:
@Araq
It never works(there is a serious bug in testament)
see https://github.com/nim-lang/Nim/pull/16567#issuecomment-753681785